crates/re_viewer/src/ui/selection_history_ui.rs

```diff
@@ -42,12 +42,17 @@ impl SelectionHistoryUi {
                     item_collection_to_string(blueprint, &previous.selection),
                 ));
 
+            let mut return_current = false;
             let response = response.context_menu(|ui| {
                 // undo: newest on top, oldest on bottom
                 for i in (0..history.current).rev() {
                     self.history_item_ui(blueprint, ui, i, history);
                 }
+                return_current = true;
```

Comment:
this will return the current selection even if the user does not click an item in the list - is this intended?

Reply:
Yeah, selecting the current thing is a no-op.

We could add an extra check just in case I guess :shrug: